src/main/java/com/faforever/client/api/FafApiAccessorImpl.java

```diff
@@ -192,7 +189,6 @@ public AchievementDefinition getAchievementDefinition(String achievementId) {
   @Override
   @Cacheable(CacheNames.LADDER_1V1_LEADERBOARD)
   @SneakyThrows
-  @SuppressWarnings("unchecked")
```

Comment:
Is that really not needed? Are u sure? Is is to suppress warning about missing type safety.

Reply:
It is marked as redundant by IntelliJ